DeepSpeech.ipynb

```diff
@@ -1325,8 +1342,25 @@
     "\n",
     "        # Loop over the batches\n",
     "        for batch in range(int(ceil(batches_per_device))):\n",
+    "            if do_fulltrace:\n",
+    "                loss_run_metadata = tf.RunMetadata()\n",
+    "            \n",
+    "            # We track run metadata (especially for memory usage) only on loss\n",
```

Comment:
I'm wondering if this is the right place for this comment instead of more standard code comments.

Is the audience internal (our group) or external?

If the audience is internal I think it's more efficient to communicate these kind of numbers via IRC or our internal meetings. 

If the audience is external then, mentioning something like LDC93S1 is likely a mistake. External to this project LDC93S1 has a very specific meaning. It's the TIMIT Acoustic-Phonetic Continuous Speech Corpus[[1](https://catalog.ldc.upenn.edu/ldc93s1)]. The one example we train with is the single sample audio file[[2](https://catalog.ldc.upenn.edu/desc/addenda/LDC93S1.wav)] LDC supplies as an example of the TIMIT Acoustic-Phonetic Continuous Speech Corpus[[1](https://catalog.ldc.upenn.edu/ldc93s1)].